src/database/migrations/1742320602314_rename_member_user_public_key_table.ts

```diff
@@ -1,12 +1,18 @@
-import type { Kysely } from 'kysely'
-import { DB } from '@/database/types'
+import { sql, type Kysely } from 'kysely'
+import { type DB } from '@/database/types'
 
 export async function up(db: Kysely<DB>): Promise<void> {
-    await db.schema.alterTable('memberUserPublicKey').renameColumn('value', 'publicKey').execute()
-    await db.schema.alterTable('memberUserPublicKey').addUniqueConstraint('unique_user_id', ['userId']).execute()
-    await db.schema.alterTable('memberUserPublicKey').renameTo('userPublicKey').execute()
+    await db.schema.alterTable('member_user_public_key').renameTo('user_public_key').execute()
+
+    await db.schema.alterTable('user_public_key').renameColumn('value', 'public_key').execute()
+    await db.schema.alterTable('user_public_key').addUniqueConstraint('unique_user_id', ['user_id']).execute()
+
+    await db.schema
+        .alterTable('user_public_key')
+        .alterColumn('public_key', (col) => col.setDataType(sql`bytea USING public_key::bytea`))
+        .execute()
```

Comment:
migrations don't use camelcase, they have to match what the tables/fields are in the DB